scripts/benchmarks/build-compare.sh

```
@@ -0,0 +1,77 @@
+# Compares Next.js build output between origin/dev and the current branch.
+# Runs two full builds — expect ~5-10 minutes total.
+set -euo pipefail
+
+REPO_ROOT="$(git rev-parse --show-toplevel)"
+CURRENT_BRANCH="$(git rev-parse --abbrev-ref HEAD)"
+NEXT_DIR="$REPO_ROOT/apps/nextjs"
+OUT_DIR="$REPO_ROOT/.bench"
+mkdir -p "$OUT_DIR"
+
+capture_metrics() {
+  local label="$1" out="$OUT_DIR/$label"
```

Shellcheck (0.11.0)

[warning] This assignment is used again in this 'local', but won't have taken effect. Use two 'local's.

(SC2318)